src/screens/ConnectScreen.js

```diff
@@ -1,17 +1,249 @@
+import { StyleSheet, Text, View, Modal } from 'react-native';
+
+import InfoModal from '../components/InfoModal';
+import ConnectPin from '../components/ConnectPin';
+import ConnectBox from '../components/ConnectBox';
 
 export default class ConnectScreen extends React.Component {
 
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isHelpModalOpen: true,
+      veterans: this.getVeterans(),
+      parterOrgs: this.getParterOrgs(),
+      activeConnection: null,  // Indicates if a veteran/org has been focused
+    }
+
+    this.onRegionChange = this.onRegionChange.bind(this);
+    this.closeHelpModal = this.closeHelpModal.bind(this);
+    this.closeConnectBox = this.closeConnectBox.bind(this);
+  }
+
+  /**
+   * Placeholder until backend data is collected.
+   */
+  getVeterans() {
+    return [
+      {
+        id: 15,
+        name: 'Ken Chen',
+        roles: ['veteran', 'caretaker'],
+        email: 'kenchen@berkeley.edu',
+        bio: 'Hi! I am a former Signal Corps officer currently residing in Berkeley, CA. Looking for assistance with finaces and benefits.',
+        lat: 37.78825,
+        lng: -122.4324,
+      },
+      {
+        id: 16,
+        name: 'James Chen',
+        roles: ['family_member'],
+        email: 'kenchen@berkeley.edu',
+        bio: 'Hi! I am a former Signal Corps officer currently residing in Berkeley, CA. Looking for assistance with finaces and benefits.',
+        lat: 37.77825,
+        lng: -122.4123,
+      },
+      {
+        id: 17,
+        name: 'Sarah Chen',
+        roles: ['combat_veteran', 'post_911'],
+        email: 'kenchen@berkeley.edu',
+        bio: 'Hi! I am a former Signal Corps officer currently residing in Berkeley, CA. Looking for assistance with finaces and benefits.',
+        lat: 37.78240,
+        lng: -122.4344,
+      },
+      {
+        id: 18,
+        name: 'Alice Chen',
+        roles: ['combat_veteran', 'caretaker'],
+        email: 'kenchen@berkeley.edu',
+        bio: 'Hi! I am a former Signal Corps officer currently residing in Berkeley, CA. Looking for assistance with finaces and benefits.',
+        lat: 37.78434,
+        lng: -122.4354,
+      },
+    ]
+  }
+
+  /**
+   * Placeholder until backend data is collected.
+   */
+  getParterOrgs() {
+    return [
+      {
+        id: 15,
+        name: 'Veterans 360',
+        email: 'kenchen@berkeley.edu',
+        lat: 37.78354,
+        lng: -122.4224,
+      },
+    ]
+  }
+
+  openHelpModal() {
+    this.setState({ isHelpModalOpen: true });
+  }
+
+  closeHelpModal() {
+    this.setState({ isHelpModalOpen: false });
+  }
+
+  openConnectBox(connection) {
+    this.setState({ activeConnection: connection });
+  }
+
+  closeConnectBox() {
+    this.setState({ activeConnection: null });
+  }
+
+  getInitialRegion() {
+    return {
+      latitude: 37.78825,
+      longitude: -122.4324,
+      latitudeDelta: 0.0922,
+      longitudeDelta: 0.0421,
```

Comment:
instead of hard coding these, will it be based off of the veteran's location when we get that from the backend?

Reply:
yea the deltas might be hard coded still but the lat/lon we will def use the user location